### ISSUE TITLE
Change Icon Set

### ISSUE THREAD

**Issue by DeveloperMal** [DeveloperMal/wger#2]
The current icon set looks outdated and they are  inconsistent with what there functions are. I suggest upgrading to a modern icon set like Ion.

**Comment by rolandgeider** [comment]
Hi there! I just saw that you opened some issues here. I agree that the iconset looks a bit outdated and doesn't really fit with the flat design of the rest of the page anymore (this was a bit different in the beginning). If you want, we can work on this together

**Comment by DeveloperMal** [comment]
Hey, thanks for checking out some of my issues. I made them to keep notes to myself separate from the main repository's issues. I see you did some work on this one. What are your ideas?

**Comment by rolandgeider** [comment]
yeah, what I have done over the weekend was replace the svg icons with new ones from font awesome, which was already used in the application. I actually only wanted to test if this was something that could work but I liked the result so much that I got carried over and replaced all icons... If you have a local instance, you can just pull from master, I haven't gotten around to updating wger.de

As to what's left to do, I'd say take a look and if you still find it ugly, then let's see what needs to be changed (I'm not a design person, so I'm thankful for any feedback) :)
